src/common/ViewerProps.ts

```diff
@@ -21,10 +16,15 @@ export interface ViewerConfig {
   redirectUri: string;
 }
 
-/**
- * Returns a list of RPCs supported by this application
- */
-export function getSupportedRpcs(): RpcInterfaceDefinition[] {
+// ipc channel name for this application
+export const dtsChannel = "dts";

```

Comment:
dts = Desktop Starter? That is not immediately obvious to me.

Reply:
yeah, I copied this from "Display Test App" where someone used "dta". Probably should be "starterChannel" or something.

Reply:
fixed